### PR TITLE
Reset password

### PR DESCRIPTION
**Pull request by shirleyhuang93**
Forget password link will redirect to reset password page.

### CODE REVIEW

**Comment by pranaygp** [comment]
Doesn't this also need a new route to be added?

File: stylesheets/resetPassword.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,200,300,700,900&subset=latin,latin-ext);
```
Comment (pranaygp) [inline stylesheets/resetPassword.css]: This file definitely doesn't need all those different font weights (400, 200, 300, 700). The usage below suggest that you're only using weight 400 so you should be able to get rid of the rest (fast loading time)

**Comment by pranaygp** [comment]
If this PR is ready, @shirleyhuang93 feel free to merge it in

**Comment by haniawni** [comment]
Hi! Y'all are rad for continuing to build CT software! Thank you @shirleyhuang93 for the implementation!

After this is merged and the Docker image updates, I should be able to deploy it here at VT, correct? It would only require auth by email at that point?

**Pull request merged**